src/components/Loader/loader.module.scss

```diff
@@ -37,8 +37,7 @@
     }
 
     &-rtl {
-      margin-left: $space-xxs;
-      margin-right: unset;
+      margin: 0 $space-xxxs;
     }
```

Comment:
seemingly this rule becomes unnecessary

Reply:
Aiming to keep the rtl reference in css as it may be needed in the future.